workflows/src/test/kotlin/hk/edu/polyu/af/bc/badge/assertion/GetAssertionTest.kt

```diff
@@ -0,0 +1,77 @@
+package hk.edu.polyu.af.bc.badge.badgeclass
+import hk.edu.polyu.af.bc.badge.UnitTestBase
+import hk.edu.polyu.af.bc.badge.flows.assertion.GetAssertionByBadgeClass
+import hk.edu.polyu.af.bc.badge.flows.assertion.IssueAssertion
+import hk.edu.polyu.af.bc.badge.flows.assertion.GetAssertionByID
+import hk.edu.polyu.af.bc.badge.flows.assertion.GetAssertionByRecipient
+import hk.edu.polyu.af.bc.badge.flows.badgeclass.CreateBadgeClass
+import hk.edu.polyu.af.bc.badge.getOrThrow
+import hk.edu.polyu.af.bc.badge.output
+import hk.edu.polyu.af.bc.badge.states.Assertion
+import hk.edu.polyu.af.bc.badge.states.BadgeClass
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.slf4j.LoggerFactory
+import kotlin.test.assertEquals
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class GetAssertionTest : UnitTestBase() {

```

Comment:
For common "set up" operations in each individual test method, you can avoid duplicated code by moving it into a `@BeforeAll` setup method. In your case, you need a `BadgeClass` for every method - just create a `BadgeClass` before all tests.